### ISSUE TITLE
Create a script to convert JSON to NOC configuration.

### ISSUE THREAD

**Issue by jjr840430** [riboseinc/libnereon#27]
Refer [issue](https://github.com/riboseinc/rvc/issues/239) from `rvc`

**Comment by ronaldtse** [comment]
I'm encountering this issue when running `json2noc`. @jjr840430 can you please fix?

```
json2noc -i /opt/rvc/etc/rvd.json  -o /opt/rvc/etc/rvd.noc
json2noc(82578,0x7fffabdd2380) malloc: *** error for object 0x7ffeee83d8f5: pointer being freed was not allocated
*** set a breakpoint in malloc_error_break to debug
Abort trap: 6
```

**Comment by jjr840430** [comment]
@ronaldtse Fixed by #30
